README.rst

```diff
@@ -25,7 +25,7 @@ On some networks discovery doesn't work reliably, in that case if you can find t
 .. code-block:: python
 
     >>> import pywemo
-    >>> url = pywemo.setup_url_for_address("192.168.1.192", None)
+    >>> url = pywemo.setup_url_for_address("192.168.1.192")
```

Comment:
WDYT about changing the documentation after the next release? That way the examples match what folks are able to download from pypi.

Reply:
Oops, see reply in main comments.